SportHub/src/modules/AdminMainArticlesSection/components/AdminMainArticlesSection.js

```diff
@@ -0,0 +1,117 @@
+import React, {useEffect, useState} from 'react'
+import styles from '../styles/style.module.scss'
+import AddMainArticle from "./AddMainArticle"
+import MAX_MAIN_ARTICLES_AMOUNT from "../constants/MaxMainArticlesAmount"
+import LableForModules from "../../../ui/LabelForModules"
+
+export default function AdminMainArticlesSection({setButtons}){
+
+    const [mainArticles, setMainArticles] = useState([])
+    const [content, setContent] = useState([])
+    const [languageId, setLanguageId] = useState(1)
+
+    useEffect(() => {
+        setMainArticles([])
+        setContent([])
+    }, [languageId])
+
+    useEffect(() => {
+        GenerateArticlesForms()
+    }, [mainArticles])
+
+    const SaveChanges = () => {
+        let jsonPostRequest = []
+        for (let articleCount = 0; articleCount < mainArticles.length; articleCount++)
+            jsonPostRequest.push({
+                                OrderNumber: mainArticles[articleCount].order,
+                                ArticleId: mainArticles[articleCount].article.id,
+                                LanguageId: 1})
+        console.log(JSON.stringify(jsonPostRequest))
+        return JSON.stringify(jsonPostRequest)
+    }
+
+    const CancelChanges = () =>{
+        setMainArticles([])
+        setContent([])
+    }
+
+    useEffect(() =>{
```

Comment:
Move useEffect under others useEffects (it will be easier to read and understand)